superset.yaml

```diff
@@ -0,0 +1,109 @@
+package:
+  name: superset
+  version: 4.0.0
+  epoch: 0
+  description: Data Visualization and Data Exploration Platform
+  copyright:
+    - license: Apache-2.0
+  options:
+    no-depends: true
```

Comment:
Could you add a comment about why we do this here?

Reply:
Will do.